cmd/bom/cmd/root.go

```diff
@@ -62,6 +62,7 @@ func init() {
 	rootCmd.AddCommand(generateCmd)
 	rootCmd.AddCommand(documentCmd)
 	rootCmd.AddCommand(versionCmd)
+	AddValidate(rootCmd)
```

Comment:
I noticed that this deviates from the pattern of the other commands. Do you plan on refactoring the others to follow this new pattern?

Reply:
Yep, tried to switch to the pattern @justaugustus recommended here: https://github.com/kubernetes-sigs/bom/pull/27#pullrequestreview-860236487
I will refactor the rest of the CLI to match.

Reply:
👍 nice